packages/sequelize-transparent-cache/lib/cache/util.js

```diff
@@ -24,6 +29,21 @@ function dataToInstance (model, data) {
   return instance
 }
 
+function loadAssociations (model) {
+  let associations = []
+  Object.keys(model.associations).forEach((key) => {
```

Comment:
As far as I understand, model.associations is always has all model associations. What if I'm only want to cache one of them, not all?

Reply:
At this point it will cache whatever association you loaded while fetching the data directly from the DB ('first non cache request'). 

Afterwards even if you build the instances with all its associations it won't make any difference or error. Only the ones that were actually cached on the initial request will be there. 

So in a way we could say that caching specifically what the user requested is already handled. 

Let me know if you had something else in mind or if I'm missing something